var/spack/repos/builtin/packages/nco/package.py

```diff
@@ -63,7 +85,11 @@ class Nco(AutotoolsPackage):
 
     def configure_args(self):
         spec = self.spec
-        return [f"--{'enable' if '+doc' in spec else 'disable'}-doc"]
+
+        return [
+            f"--{'enable' if ' doc' in spec else 'disable'}-doc",
+            f"--{'enable' if ' openmp' in spec else 'disable'}-openmp",
```

Comment:
This is not the standard way to set configure arguments for variants (see https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#enable-or-disable). The API for the helpers changed while back to return a list of strings but you can still use them.

For example, since both are boolean variants, you could use:

```suggestion
            f"{self.enable_or_disable('+doc')[0]}-doc",
            f"{self.enable_or_disable('+openmp')[0]}-openmp",
```

Reply:
@altheaden, could you make the suggested changed?  It looks good to me.